Marlin/Conditionals.h

```diff
@@ -46,6 +46,21 @@
 
   #define CONFIGURATION_LCD
 
+  #if ENABLED(CartesioUI)
+    #define DOGLCD
+    #define SDSUPPORT
+    #define ULTIPANEL
+    #define NEWPANEL
+    #define DEFAULT_LCD_CONTRAST               90
+    #define ENCODER_PULSES_PER_STEP            2
+    #define ENCODER_STEPS_PER_MENU_ITEM        1
+    #define SPEAKER
```

Comment:
Don't define `SPEAKER` here. Users set this option in `Configuration.h` if they want it, or they can disable it there.